authors/apps/profiles/serializers.py

```diff
@@ -23,12 +23,51 @@ def get_username(self, obj):
 
 
 class ProfileUpdateSerializer(serializers.ModelSerializer):
+    username = serializers.SerializerMethodField(source="User")
     class Meta:
+        extra_kwargs ={
+            'username':{
+                'read_only':True
+            }
+        }
         model = Profile
         fields = [
+            'username',
             'first_name',
             'last_name',
             'bio',
             'image',
             'date_of_birth'
         ]
+   
+    
+    def get_username(self, obj):
+        return f"{obj.user.username}"
+
+    def to_representation(self, instance):
+        print("instance", instance)
```

Comment:
@armstrongsouljah remove print statement and document function

Reply:
Thanks for spotting that. Let me resolve it right away.